app/build.gradle

```diff
@@ -73,7 +73,7 @@ android {
 
     compileOptions {
         // Flag to enable support for the new language APIs
-        coreLibraryDesugaringEnabled true
+        coreLibraryDesugaringEnabled false
```

Comment:
@Isira-Seneviratne do you have any idea why the build works without issues even if this is set to `false`? How can Java methods be desugared if this is missing?

Reply:
Build works, but running the app on API 19 does not. We need to revert this